code/smartPointers/Makefile

```diff
@@ -5,7 +5,7 @@ clean:
 	rm -f *o *so smartPointers *~ smartPointers.sol
 
 % : %.cpp
-	$(CXX) -g -std=c++17 -Wall -Wextra -o $@ $<
+	$(CXX) -g -std=c++20 -Wall -Wextra -o $@ $<
 
 %.sol : solution/%.sol.cpp
-	$(CXX) -g -std=c++17 -Wall -Wextra -o $@ $<
+	$(CXX) -g -std=c++20 -Wall -Wextra -o $@ $<
```

Comment:
You may want to apply the same changes to `CMakeLists.txt` as well.

Reply:
Done, all green. Merging now.